SpringProject/src/main/java/com/gradproject2019/Controller.java

```diff
@@ -1,13 +1,20 @@
 package com.gradproject2019;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
 @org.springframework.stereotype.Controller
```

Comment:
Probably makes sense to import this and use just the simple @Controller annotation